packages/stack/src/views/Header/Header.tsx

```diff
@@ -4,64 +4,56 @@ import HeaderSegment from './HeaderSegment';
 import { HeaderProps, HeaderTitleProps } from '../../types';
 import HeaderTitle from './HeaderTitle';
 
-export default class Header extends React.PureComponent<HeaderProps> {
-  render() {
-    const {
-      scene,
-      previous,
-      layout,
-      navigation,
-      styleInterpolator,
-    } = this.props;
-    const { options } = scene.descriptor;
-    const title =
-      typeof options.headerTitle !== 'function' &&
-      options.headerTitle !== undefined
-        ? options.headerTitle
-        : options.title !== undefined
-        ? options.title
-        : scene.route.name;
+export default React.memo(function(props: HeaderProps) {
```

Comment:
We need to give the component a name so that it shows up in devtools:

```suggestion
export default React.memo(function Header(props: HeaderProps) {
```

Reply:
done 👍